source/connect/csot.txt

```diff
@@ -1,210 +1,218 @@
 .. _pymongo-csot:
 
-Client Side Operation Timeout
-=============================
+Limit Server Execution Time
+===========================
 
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 1
+   :depth: 2
    :class: singlecol
 
 .. facet::
    :name: genre
    :values: reference
  
 .. meta::
-   :keywords: error, blocking
+   :keywords: error, blocking, thread, task
 
-PyMongo 4.2 introduced the ``~pymongo.timeout()`` method and the ``timeoutMS``
-URI and keyword argument to the ``~pymongo.mongo_client.MongoClient`` class.
-These features allow applications to more easily limit the amount of time that
-one or more operations can execute before control is returned to the app. This
-timeout applies to all of the work done to execute the operation, including
-server selection, connection checkout, serialization, and
-server-side execution.
+Overview
+--------
 
-Basic Usage
------------
+With {+driver-short+}, you can specify a **client side operation timeout**, a time limit
+for server operations to run. The timeout applies all steps needed to complete the
```

Comment:
The phrasing here initially made me think that it was saying that a time limit is something that a server can run (like an operation or something). Maybe something like this will be more clear:

```suggestion
With {+driver-short+}, you can specify a **client-side operation timeout**, which specifies a time limit
for how long a server can run operations. The timeout applies all steps needed to complete the
```

Reply:
good call